neurodegeneration/processing-containers/normative_biomarker_visualizer/files/normative_biomarker_visualizer.py

```diff
@@ -258,7 +273,7 @@ def plotWithRef(dfRef, WMLSref, dfSub, fname, spareAD, spareBA):
 
 # TODO: 
 def makeFlagTablePKL(dfRef,mydict,MUSErois,path):
-	maphemi = pd.read_csv('/refs/MUSE_ROI_Dictionary.csv')
+	maphemi = pd.read_csv('../refs/MUSE_ROI_Dictionary.csv')
```

Comment:
Make sure the path works after creating docker container.

Reply:
done